### DISCUSSION TITLE
No instructions for uninstalling on macOS?

### DISCUSSION THREAD

**Discussion by jc-dgit** [microsoft/openjdk#77] [Q&A] [answered]
Just installed the MS OpenJDK (.pkg) on macOS, as I need something that can do Java Web Start for access to an ancient HP iLO console.

Turns out HP iLO doesn't want to work with it, and I need to install a different (AdoptOpenJDK or old Oracle) version instead.

So, went to uninstall the MS OpenJDK... and there seems to be no way to do it? ☹️

Looked for info in these places so far:

* The MS website (https://docs.microsoft.com/en-us/java/openjdk/)
* The FAQ on the same site (zero info)
* The front page of this GitHub repo (again, no info)
* The "Discussions" area for this repo (no info)
* The wiki for this repo... oh wait, there isn't one

What the.... ? 😦 

Ok, can someone please advise how to uninstall this?  **Surely** that's a common scenario.

**Comment by jc-dgit** [comment, marked as answer]
For anyone else who hits the same problem, this should work:

```
$ sudo rm -rf /Library/Java/JavaVirtualMachines/microsoft-11.jdk
$ sudo pkgutil --forget com.microsoft.11.jdk
```

That should delete the MS OpenJDK 11 files, and remove the entry for it from the macOS pkg database.

**Reply by karianna** [reply]
@brunoborges - Something worth adding to our docs?
